imports/ui/components/reader.js

```diff
@@ -624,10 +724,35 @@ scrollText = function(){
 	if (chronologicalReading) {
 		$('#textColumn').scrollTop($('#textColumn')[0].scrollHeight);
 	}else{
-		$('#textColumn').scrollTop($('#textColumn')[0])	
+		$('#textColumn').scrollTop($('#textColumn')[0])
 	}
 }
 
+const isOverflown = ({ clientHeight, scrollHeight }) => {
```

Comment:
maybe add one or two lines of commentary to explain what this function is for
remove console.logs

Reply:
it's done.